### ISSUE TITLE
Highlighting bytes causes crash

### ISSUE THREAD

**Issue by webdestroya** [HexFiend/HexFiend#354]
Highlighting text for this specific sequence always causes an immediate crash.

## Steps
* Open Hex Fiend
* Paste in the data below
* Data helpers?(The things on the bottom)
  * Signed Int: le, dec
  * Signed Int: be, dec
  * Floats: le
  * UTF-8
  * Binary
  * Unsigned Int: be, dec
* Highlight the first 4 bytes. (Things are fine)
* Start highlighting from byte 5
* CRASH after about 8 or 10 bytes

## Data
```
4200000025007465737470726570000001000000010000000946616c6c20323032300001000044000000065000450000000900000000005300000004
```

## Crash Report
[Hex Fiend-2021-12-10-173121.ips.zip](https://github.com/HexFiend/HexFiend/files/7696190/Hex.Fiend-2021-12-10-173121.ips.zip)

## System info
* Version: 2.14.1
* Hardware: M1 Max, Monterey. arm64

## Exception?
```
Application Specific Information:
Crashing on exception: NSConcreteAttributedString initWithString:: nil value
```

**Comment by webdestroya** [comment]
Removing the "Float" preview stops the crashing.

**Comment by kainjow** [comment]
Fixed with #345 320
